modules/passkey/contracts/ProxySimulator.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+/* solhint-disable no-complex-fallback */
+pragma solidity >=0.8.0;
```

Comment:
move this file to certora folder under helpers

Reply:
moved